datahub-web-react/src/app/preview/DefaultPreviewCard.tsx

```diff
@@ -329,12 +335,14 @@ export default function DefaultPreviewCard({
                         </NoMarkdownViewer>
                     </DescriptionContainer>
                 )}
-                {(domain || hasGlossaryTerms || hasTags) && (
+                {(dataProduct || domain || hasGlossaryTerms || hasTags) && (
```

Comment:
Oh boy this is wild

Reply:
yeah this component could use some TLC tbh